_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,214 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles.
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope), except if the element has an implicit semantic role that is identical to its explicit semantic role.
+
+### Expectation
+
+The target element is [owned by](#owned-by) an element that has a [semantic role](#semantic-role) matching one of the [WAI-ARIA required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) for the target element.
+
+**Note:** The definition of [owned by](#owned-by) used in this rule is diverging from the definition of ["owned element" in WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element). See more in the [owned by](#owned-by) definition.
+
+## Assumptions
+
+This rule assumes that the explicit [semantic role](#semantic-role) on the target element is being used with the intention to comply to WCAG. If the explicit semantic role on the target element is incorrectly used, and any relationships between elements are already programmatically determinable, failing this rule might not result in accessibility issues for real users, and it should then not be considered a failure under WCAG success criterion 1.3.1 Info and Relationships.
+
+## Accessibility Support
+
+This rule relies on assistive technologies to recognize which elements are [owned by](#owned-by) each other. This includes when the element is owned by another element that is an ancester, but not a parent of the target element. Some assistive technologies does not accept these `owned by` relationships, unless workarounds are used.
```

Comment:
`ancester` -> `ancestor`. The last "owned by" should not be in backticks, which is normally used for conveying code.